src/arrays.js

```diff
@@ -1,61 +1,66 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length
```

Comment:
There is a couple of things to mention here.
I am not sure what is that you are trying to do, but here you are using the remainder assignment (%=) operator which divides a variable by the value of the right operand and assigns the remainder to the variable.
Which means first you would have to declare a variable which is allowed to be reassigned to do that you have to use the let declaration.

for example:
`````
let a = 3;
console.log(a %= 2);
// Expected output: 1
`````

But here we have index as a parameter provided, this can not be reassigned.

You are on the right way to get to the solution though, almost there :)
You just have to remove the assignment part and just use the remainder and you can use this directly in the square brackets []

`````
const getNthElement = (index, array) => {
  return array[index % array.length];
};
`````